docs/DeveloperGuide.md

```diff
@@ -85,14 +85,14 @@ Guarantees:
 Use case ends.
 
 **Extensions:**
-1a. User enters an illegal command.  
-&ensp;&ensp;1a1. System warns that the <u>[command is illegal (UC07)](#UC07-Warn-on-Illegal-Command)</u>.  
+1a. User enters an illegal command.
+&ensp;&ensp;1a1. System warns that the <u>[command is illegal (UC07)](#UC07-Warn-on-Illegal-Command)</u>.
 &ensp;&ensp;1a2. User acknowledges and closes the warning.
 
 Use case ends.
 
 ### UC02: Delete a task
-Actor(s): Project Manager  
+Actor(s): Project Manager
```

Comment:
I might be wrong but removing the double space might cause `Guarantee` to be at the same line as `Project Manager`